python/GafferUI/Slider.py

```diff
@@ -531,7 +531,7 @@ def __emitValueChanged( self, reason ) :
 
 	def __eventValue( self, event ) :
 
-		f = event.line.p0.x / float( self.size().x )
+		f = ( event.line.p0.x - 0.5 ) / float( self.size().x )
```

Comment:
Isn't this unnecessary as well? Imagine clicking on the middle pixel of a slider with `width = 3`. Don't we want `1.5 / 3` to give us `0.5`?

Reply:
After talking through this offline, yes, the offset should not be removed here. I double checked and there are no problems getting the slider of a spline shader all the way down to 0.0 and up to 1.0.